packages/git/src/browser/dirty-diff/dirty-diff-decorator.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { inject, injectable } from 'inversify';
+import {
+    EditorManager, EditorDecorationsService, Range, Position, DeltaDecoration,
+    ModelDecorationOptions, OverviewRulerLane
+} from '@theia/editor/lib/browser';
+import { DityDiffUpdate } from './dirty-diff-manager';
+import { LineRange } from './diff-computer';
+
+export enum DirtyDiffDecorationType {
+    AddedLine = 'dirty-diff-added-line',
+    RemovedLine = 'dirty-diff-removed-line',
+    ModifiedLine = 'dirty-diff-modified-line',
+}
+
+const AddedLineDecoration = <ModelDecorationOptions>{
+    linesDecorationsClassName: 'dirty-diff-glyph dirty-diff-added-line',
+    overviewRuler: {
+        color: 'rgba(0, 255, 0, 0.6)',
```

Comment:
Could we use the colors we use in the git views? Also, it would be good to rely on colors from the theme.
So theme authors can adjust them.

Reply:
let me check, if we have proper alternatives for the colors.

Reply:
in case, I'm missing something, which git view colors do you mean? I understand the point of theming, and I changed to variables form styles. the issue is again, that these are not rgba values, which should be used preferably for decorations in monaco, as they might be layered. also, it looks like those opaque markers are rendered grayish in the overview ruler. 

<img width="375" alt="screen shot 2018-02-27 at 16 03 39" src="https://user-images.githubusercontent.com/914497/36736020-d5579312-1bd7-11e8-9baf-5723207e4e27.png">
<img width="361" alt="screen shot 2018-02-27 at 16 03 19" src="https://user-images.githubusercontent.com/914497/36736021-d574228e-1bd7-11e8-9a8b-0e8d69b5b4dd.png">

Reply:
I double-checked that, and it's true, only rgba values are rendered nicely in the overview ruler.

Reply:
I'd like to add status colors to the themes. WDYT @svenefftinge ?